Library/src/CSP/Systems/EventTicketing/EventTicketing.cpp

```diff
@@ -20,16 +20,56 @@
 
 namespace chs = csp::services::generated::aggregationservice;
 
+csp::systems::EventTicketingVendor VendorNameToEnum(const csp::common::String& VendorName)
+{
+	csp::systems::EventTicketingVendor Vendor = Vendor = csp::systems::EventTicketingVendor::Unknown;
+	if (VendorName == "eventbrite")
+	{
+		Vendor = csp::systems::EventTicketingVendor::Eventbrite;
+	}
+	else
+	{
+		FOUNDATION_LOG_MSG(csp::systems::LogLevel::Warning,
+						   "Encountered an unknown ticketing vendor string when parsing a response from services. Defaulting to 'Unknown'");
+		Vendor = csp::systems::EventTicketingVendor::Unknown;
```

Comment:
Could remove this since it's set initially to this.

Reply:
Yep, happy with that.

Reply:
Pushed the change to the branch.